cli/src/cmd_auth.rs

```diff
@@ -130,6 +137,7 @@ pub fn parse_host(input: &str) -> Result<url::Url> {
 #[derive(Parser, Debug, Clone)]
 #[command(verbatim_doc_comment)]
 pub struct CmdAuthLogin {
+    // TODO seems pointless
     /// Read token from standard input.
     #[clap(long)]
     with_token: bool,
```

Comment:
Sadly, we can't stop people from grabbing a token from a machine and pasting it to the credentials file of another. By passing the token they already have through this flag we can deter them from manually editing the credentials file.